### PR TITLE
Amp 80403 add supported versions of rn rn cli for rn sdk releases

### PR DESCRIPTION
**Pull request by justin-fiedler**
## Description

* Add [compatibility matrix of supported versions of RN, RN CLI, Gradle](https://pr-863.d19s7xzcva2mw3.amplifyapp.com/data/sdks/react-native/#compatibility-matrix) for ReactNative SDK
* Fixed [layout of troubleshooting list](https://pr-863.d19s7xzcva2mw3.amplifyapp.com/data/sdks/react-native/#troubleshooting)

https://pr-863.d19s7xzcva2mw3.amplifyapp.com/data/sdks/react-native/

Related to
*  #856
* https://github.com/amplitude/Amplitude-ReactNative/issues/95
* https://github.com/amplitude/Amplitude-ReactNative/issues/168

### CODE REVIEW

**Review by bgiori — APPROVED** [review]
React native *shakes fist*

This is great info though 👍

File: docs/data/sdks/react-native/index.md [hunk 1]
```diff
@@ -16,9 +16,20 @@ This is the official documentation for the Amplitude Analytics React Native SDK.
...
     [:material-github: GitHub](https://github.com/amplitude/Amplitude-ReactNative) · [:material-code-tags-check: Releases](https://github.com/amplitude/Amplitude-ReactNative/releases) · [:material-book: API Reference](https://amplitude.github.io/Amplitude-ReactNative/)
 
+--8<-- "includes/ampli-vs-amplitude.md"
+
 --8<-- "includes/sdk-rn/rn-notification.md"
 
---8<-- "includes/ampli-vs-amplitude.md"
+## Compatibility Matrix
+
+The following matrix lists the support for Amplitude React Native SDK version for different versions of React Native and React Native cLI.
```
Comment (yuhao900914) [inline docs/data/sdks/react-native/index.md:25]: Typo: CLI.

**Review by yuhao900914 — APPROVED** [review]

**Comment by Mercy811** [comment]
How about including more related versions, like Android API version and Kotlin version in [this issue](https://github.com/amplitude/Amplitude-ReactNative/issues/165)?

**Comment by justin-fiedler** [comment]
> How about including more related versions, like Android API version and Kotlin version in [this issue](https://github.com/amplitude/Amplitude-ReactNative/issues/165)?
> 
> And another one for TS RN?

Thanks, unfortunately I don't know the exact versions we support. I think it's best to wait until it comes up and then update docs after investigation.

**Pull request merged**